### DISCUSSION TITLE
面板中日志显示本地ip地址回环了

### DISCUSSION THREAD

**Discussion by rsgdn** [nikkinikki-org/OpenWrt-nikki#151] [Q&A] [answered]
![屏幕截图 2024-09-01 094209](https://github.com/user-attachments/assets/fe5537df-c61e-4590-ab5a-3e267c549603)
这个是核心得问题还是加入规则能解决？就正常使用中一直出现🙏

**Comment by morytyann** [comment, marked as answer]
TUN模式？有没有配置好`代理配置 -> 绕过 -> WAN6 接口`

**Reply by rsgdn** [reply]
> TUN模式？没有好配置`代理配置 -> 绕过 -> WAN6 接口`

不是TUN模式，是TPROXY模式，然后按你设置绕过wan6接口就没事了，wan6接口要把两个都选上才行，谢谢🙏
